package.json

```diff
@@ -71,6 +71,7 @@
     "react-dom": "^17.0.2",
     "react-helmet": "^6.1.0",
     "react-intl": "^5.25.1",
+    "react-refresh": "^0.14.2",
```

Comment:
Not part of this pr

Reply:
### When I ran the application it showed this -
**ERROR in polyfill
Module not found: Error: Cannot find module 'react-refresh'**

Reply:
Can you suggest what can be done ?

Reply:
I can't replicate that issue, what's your browser/system/node version etc...?

Reply:
browser - chrome
system - windows 11
node - v19.6.1